README.md

```diff
@@ -1968,6 +1968,27 @@ ANSWERS.addComponent('Map', {
 };
 ```
 
+When using MapBox (`mapBox`) as the `mapProvider`, the valid locale options
```

Comment:
What do you think about an h3 section titled "Supported Map Locales" and creating a note in the locale option on line 1931 that says "Refer to the section "Supported Map Locales" below for the list of supported locales'. It might even be possible to make a clickable link to it

Reply:
Sounds good!